recurly/base_client.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, api_key):
         self.__api_key = api_key
         self.__conn = http.client.HTTPSConnection(HOST, PORT)
 
-    def _make_request(self, method, path, body, params):
+    def _make_request(self, method, path, body, **options):
```

Comment:
We probably want to validate the `options` object. I'll make a follow up ticket for it.